### PR TITLE
Add feedback form

### PR DESCRIPTION
**Pull request by bhargav265**
Closes #X

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by bhargav265** [comment]
[React Issue 2: Add FeedbackStore, and add FeedbackForm](https://trello.com/c/fdALnjGR/13-react-issue-2-add-feedbackstore-and-add-feedbackform)

File: app/javascript/image_sharing/components/FormComponent.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                 value={this.props.stores.feedbackStore.name}
+                 onChange={this.updateName}
+        rowClassName = "name"/>
+        <FormRow type="textarea"
+                 label="Comments"
+                 value={this.props.stores.feedbackStore.comment}
+                 onChange={this.updateComment}
+                 rowClassName = "comment"
+        />
+        <Button color="primary" disabled={false} outline={false}>
```
Comment (butzhang) [inline app/javascript/image_sharing/components/FormComponent.js]: delete `disabled={false} outline={false}`

File: app/javascript/image_sharing/stores/FeedbackStore.js [hunk 2]
```diff
@@ -1,5 +1,19 @@
+import { action, observable } from 'mobx';
+
 export class FeedbackStore {
   /* Implement your feedback store*/
+  @observable name = '';
```
Comment (butzhang) [inline app/javascript/image_sharing/stores/FeedbackStore.js]: add test for it

File: app/javascript/image_sharing/test/components/FormComponent.spec.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      props = {
+        stores: {
+          feedbackStore: {
+            updateName: sinon.spy(),
+            updateComment: sinon.spy()
+          }
+        }
+      }
+      sandbox = sinon.createSandbox();
+
```
Comment (butzhang) [inline app/javascript/image_sharing/test/components/FormComponent.spec.js]: extra line here

File: app/javascript/image_sharing/components/FormComponent.js (hunk 1 above)
Comment (butzhang) [inline app/javascript/image_sharing/components/FormComponent.js]: and put the button in the center

**Pull request merged**